presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

```diff
@@ -763,6 +790,20 @@ public CounterStat getAuthorizationFail()
         return authorizationFail;
     }
 
+    private void systemAuthorizationCheck(Consumer<SystemAccessControl> check)
+    {
+        try {
+            for (SystemAccessControl systemAccessControl : systemAccessControls.get()) {
+                check.accept(systemAccessControl);
+                authorizationSuccess.update(1);
```

Comment:
shouldn't this be outside of the loop?

Reply:
i think it should

Reply:
btw some successes/failures are counted twice, because `authorizationCheck` wraps calls to `systemAuthorizationCheck` and both update the same counter

(it seems preexisting)

Reply:
Changed it